impeller/renderer/backend/gles/proc_table_gles.h

```diff
@@ -75,6 +76,12 @@ struct GLProc {
   auto operator()(Args&&... args) const {
 #ifdef IMPELLER_DEBUG
     AutoErrorCheck error(error_fn, name);
+    // We check for the existence of extensions, and reset the function pointer
+    // but it's still called unconditionally below, and will segfault. This
+    // validation log will at least give us a hint as to what's going on.
+    if (!IsAvailable()) {
```

Comment:
If it's going to crash, it's probably best to crash here, no?

Reply:
Yeah that's a good point. Changed to a FML_CHECK.